pom.xml

```diff
@@ -1114,8 +1114,6 @@
                 <artifactId>spotless-maven-plugin</artifactId>
                 <version>${version.spotless}</version>
                 <configuration>
-                    <!-- optional: limit format enforcement to just the files changed by this feature branch -->
-                    <ratchetFrom>origin/master</ratchetFrom>
                     <formats>
```

Comment:
While this works, we want to use ratchetFrom if we can. Any clue how to make this work for forks like yours?

Reply:
sure, done